pages/index.js

```diff
@@ -83,7 +83,9 @@ const chooseEnd = ({
         end = 4
       }
     }
-  } else if (cough || soreThroatAches || agueusiaAnosmia) {
+  } else if ((cough && !soreThroatAches && !agueusiaAnosmia) ||
+	     (!cough && soreThroatAches && !agueusiaAnosmia) ||
+	     (!cough && !soreThroatAches && !agueusiaAnosmia)) {
```

Comment:
Remplacer
`(!cough && !soreThroatAches && !agueusiaAnosmia)`

par
`(!cough && !soreThroatAches && agueusiaAnosmia)`

la condition est un doublon du dernier use case
`IF NOT toux AND NOT douleurs AND NOT anosmie => FIN8`

Reply:
Corrigé dans 03e7544a03 - merci ! On aurait besoin d'un XOR, pas de `||`, car il faut un _seul_ des trois symptômes ici.

(Comme la condition précédente a aussi changé, il n'est effectivement plus logiquement nécessaire d'avoir un XOR ici... car le cas où il y a deux symptômes en même temps est traité dans cette condition précédente - mais l'algorithme a été validé tel quel par les médecins, je ne veux pas relancer une discussion sur l'économie logique de l'arbre - gardons cela.)